addons/crm/models/crm_lead.py

```diff
@@ -2446,7 +2446,7 @@ def _pls_get_lead_pls_values(self, domain=[]):
             self.flush(['probability'])
             query = """SELECT id, probability, %s
                         FROM %s
-                        WHERE %s order by team_id asc"""
+                        WHERE %s order by team_id asc, id"""
```

Comment:
If we want to be coherent with the model, should be id desc :)

Reply:
We can change that!